docs/network_firewall.md

```diff
@@ -81,7 +81,7 @@ Leave the defaults for "Time Server Information". Click Next.
 
 On "Configure WAN Interface", enter the appropriate configuration for your network. Consult your local sysadmin if you are unsure what to enter here. For many environments, the default of DHCP will work and the rest of the fields can be left blank. Click Next.
 
-For "Configure LAN Interface", set the IP address and subnet mask of the Application Subnet for the LAN interface. Click Next.
+For "Configure LAN Interface", set the IP address and subnet mask of the Application Subnet for the LAN interface.  Be sure that the CIDR prefix correctly corresponds to your subnet mask-- pfsense should automatically calculate this for you, but you should always check. Click Next.
```

Comment:
:+1: Furthermore, in L36, I suggest: `This is a very common subnet choice for home routers.` -> `The /24 subnet is a common choice for home routers.` Specifically referring to the subnet by CIDR notation adds clarity for non-technical folks, at no cost to clarity for technical audiences.